_data/banking.yml

```diff
@@ -521,14 +521,22 @@ websites:
     twitter: Holvi
     facebook: HolviCom
 
-  - name: HSBC
-    url: https://www.hsbc.com/
+  - name: HSBC UK
+    url: https://www.hsbc.co.uk/
     img: hsbc.png
     tfa:
       - proprietary
       - hardware
     doc: https://www.hsbc.co.uk/help/security-centre/secure-key/secure-key-help/
 
+  - name: HSBC US
+    url: https://www.us.hsbc.com/
+    img: hsbc.png
+    twitter: HSBC_US
+    facebook: HSBCUS
+    status: https://www.us.hsbc.com/online-banking/security-device/
+    exception: "Proprietary device password OR normal account password logs in. Only special actions require both."
```

Comment:
As their current system doesn't meet our [definition](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#a-note-on-definitions) of 2FA there's no reason to have the exception tag in the listing.
```suggestion
```